### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by alexmiddeleer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+### Upgrading EPC
+
+Upgrading the EPC to a newer version is easy if you already know how to build the EPC. Please see the docs in that repo for details. There are two ways to go about it. The first is to manually copy/paste the built files into the /epc directory. If you are adding a new beta version, just copy paste the built files into /beta/epc. The second way it to run a bash script, either `upgrade-epc.sh` or `upgrade-epc-beta.sh`. You will need `git` and `npm` available, as well as any other build dependencies of EPC. This will download either the beta or master version of epc as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules), build it, and automatically copy over the files. You will still need to use git to commit the changes, and please take this opportunity to do a final check to make sure everything worked properly. If you push to master, you have deployed to production, so be careful.
```
Comment (matt-kimball) [inline README.md]: The `upgrade-epc.sh` script will clobber the card list.  So far, @PapaCapricorn has been editing the card list on the live site, and it has needed merging back with the card list on the epc repo.  If one were to follow these instructions, you'd lose any changes made to the card list.

We need a better process for the card list merge.

Comment (alexmiddeleer) [reply README.md]: Ah right, I forgot about that. I always catch it before I commit the changes. I will add a note about this in the README unless I can think of a quick improvement.

**Comment by alexmiddeleer** [comment]
@matt-kimball Let me know what you think of the latest commit.

**Comment by matt-kimball** [comment]
> @matt-kimball Let me know what you think of the latest commit.

I'm satisfied.

**Pull request merged**